persistence.js

```diff
@@ -178,17 +180,18 @@ RedisPersistence.prototype.removeSubscriptions = function (client, subs, cb) {
   var count = 0
   var errored = false
 
-  subs.reduce(function (multi, topic) {
+  for (var i = 0; i < subs.length; i++) {
+    var topic = subs[i]
     var subClientKey = 'sub:client:' + topic
     multi.hdel(subClientKey, client.id)
     that._waitFor(client, topic, finish)
     count++
-    return multi.hdel(clientSubKey, topic)
-  }, multi)
+    multi.hdel(clientSubKey, topic)
+  }
 
   this._removedSubscriptions(client, subs.map(toSub))
 
-  multi.exec(function (err, results) {
+  multi.exec(function execMulti (err, results) {
```

Comment:
You mean inside the scope of removeSubscriptions right ? 
Because at the top level, I may not have access to the function objects